spec/presenters/sufia/collection_presenter_spec.rb

```diff
@@ -58,4 +58,9 @@
     subject { presenter.total_items }
     it { is_expected.to eq 0 }
   end
+
+  describe "#identifier" do
+    subject { presenter.identifier }
```

Comment:
incidentally, based_near and related_url are also delegated and are being tested the same way.